libraries/Bluefruit52Lib/src/bluefruit.cpp

```diff
@@ -310,7 +313,15 @@ bool AdafruitBluefruit::begin(uint8_t prph_count, uint8_t central_count)
   #error Clock Source is not configured, define USE_LFXO or USE_LFRC according to your board in variant.h
 #endif
 
+  /*------------------------------------------------------------------
+  ** BLE only Softdevices have 2-args sd_softdevice_enable()
+  ** BLE & ANT+ Softdevices have 3-args sd_softdevice_enable()
+  */
+  #ifdef ANT_LICENSE_KEY
```

Comment:
hmm, this isn't aligned well, but that is not really an issue to further delay this PR.